Source/Core/EllipsoidTerrainProvider.js

```diff
@@ -111,6 +113,18 @@ define([
         },
 
         /**
+         * Gets a promise that resolves to true when the provider is ready for use.
+         * @memberof EllipsoidTerrainProvider.prototype
+         * @type {Promise.<Boolean>}
+         * @readonly
+         */
+        readyPromise : {
+            get : function() {
+                return when.resolve(true);
```

Comment:
Same as my first comment, I won't mention it anymore, just change it everywhere you were doing this.